### PR TITLE
Bugfix/component implementation

### PR DESCRIPTION
**Pull request by gdouglas**
The block variation on the query is big enough it should be it's own plugin. I 

- removed block variations, 
- updated the linter config so it's a bit better with CSS/JS 
- Ordered pages
- Added a demo category
- fixed a button issue
- removed extra spotlight pages

### CODE REVIEW

File: patterns/components/blue_feature_blocks.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"textColor":"white"} -->
-<p class="has-white-color has-text-color">Learn more about What’s New in the Faculty of Medicine and at UBC.</p>
+<p class="has-white-color has-text-color">Learn more about What&rsquo;s New in the Faculty of Medicine and at UBC.</p>
```
Comment (daryashokouhi) [inline patterns/components/blue_feature_blocks.php:28]: Is this change intentional?

Comment (gdouglas) [reply patterns/components/blue_feature_blocks.php:28]: No but it's not that big of a deal.
